jobs/pgbackup/templates/bin/pgbackup

```diff
@@ -27,48 +22,73 @@ when 9.1, "9.1"
 when 9.0, "9.0"
   "postgres90"
 else
-  "postgres95"
+  "postgres90"
 end
 %>
 
 PGVERSION="<%= pgversion %>"
-export PGHOST PGPORT PGUSER PGPASSWORD PGVERSION
-echo "export PGHOST=$PGHOST"
-echo "export PGPORT=$PGPORT"
-echo "export PGUSER=$PGUSER"
-echo "export PGVERSION=$PGVERSION"
```

Comment:
consider restoring logs, and refine with AWS S3 bucket and endpoinbt (but not the secret key)